inventory/serializers.py

```diff
@@ -28,10 +28,18 @@ class Meta:
         fields = ["id", "username", "email", "extras"]
 
 
+class OrderItemSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Item
+        fields = "__all__"
+
```

Comment:
oops i forgot that item info is also needed in the `ItemExpiry` serialiser

but as `OrderItemSerializer` is also used in line 60, perhaps we can name this as something else, like `ItemInfoSerializer`